fe/fe-core/src/main/java/org/apache/doris/catalog/Partition.java

```diff
@@ -49,7 +48,6 @@
     private static final Logger LOG = LogManager.getLogger(Partition.class);
 
     public static final long PARTITION_INIT_VERSION = 1L;
-    public static final long PARTITION_INIT_VERSION_HASH = 0L;
```

Comment:
Why not remove field `visibleVersionHash`, `committedVersionHash` and `nextVersionHash` of this class?